v4.2.0 can't be installed because index.js is missing

The latest release v4.2.0 is broken:

```
npm ERR! path /Users/tim/[...]/node_modules/lint-staged/index.js
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/tim/[...]/node_modules/lint-staged/index.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

I think this can only be fixed by releasing a new fix version, which includes the `index.js`.

CC @okonet

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
   },
   "bin": "index.js",
   "files": [
+    "index.js",
     "src"
   ],
   "scripts": {

